Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -562,16 +585,23 @@ def __init__(self, ):
         self.messages = []  # type: list
         self.write_buf = []  # type: list
         self.replace_strs = []  # type: list
+        self.buffering = True
         # if for some reason you don't want to auto add credentails.password to replace strings
         # set the os env COMMON_SERVER_NO_AUTO_REPLACE_STRS. Either in CommonServerUserPython, or docker env
-        if (not os.getenv('COMMON_SERVER_NO_AUTO_REPLACE_STRS') and hasattr(demisto, 'getParam')
-                and isinstance(demisto.getParam('credentials'), dict)
-                and demisto.getParam('credentials').get('password')):
-            pswrd = self.encode(demisto.getParam('credentials').get('password'))
-            to_encode = pswrd
-            if IS_PY3:
-                to_encode = pswrd.encode('utf-8', 'ignore')
-            self.add_repalce_strs(pswrd, base64.b64encode(to_encode))
+        if (not os.getenv('COMMON_SERVER_NO_AUTO_REPLACE_STRS') and hasattr(demisto, 'getParam')):
+            # add common params
+            if isinstance(demisto.getParam('credentials'), dict) and demisto.getParam('credentials').get('password'):
+                pswrd = self.encode(demisto.getParam('credentials').get('password'))
+                self.add_replace_strs(pswrd, b64_encode(pswrd))
+            if demisto.getParam('api_key'):
+                s = demisto.getParam('api_key')
+                self.add_replace_strs(s, b64_encode(s))
+            if demisto.getParam('apikey'):
```

Comment:
we are missing many parameters, i think it will be better to iterate the parameters and see if they include some keywords
for example we won't catch `apiKey` or `bot_password`

Reply:
Good idea. I pushed an update.